### PR TITLE
Add providers/services seeds

### PR DESCRIPTION
**Pull request by travisvalentine**
Used the data from https://docs.google.com/spreadsheets/d/1gNjhukxXvOBlDmOD3ODczub7H1_p0hAkdRZJ1iBvBcI/edit?ts=580b86b6#gid=0

With male/female non-null constraint, for food/utility services they are just false

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,14 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+Provider.destroy_all
```
Comment (tpitale) [inline db/seeds.rb:8]: Should we do this `if ENV['RESET']`?

File: db/seeds.rb [hunk 2]
```diff
@@ -5,3 +5,14 @@
...
+services.each do |service_attributes|
```
Comment (tpitale) [inline db/seeds.rb:14]: And this `if Provider.all.empty?` (or something)

File: db/seeds.rb (hunk 1 above)
Comment (travisvalentine) [reply db/seeds.rb:8]: Fixed in https://github.com/tpitale/counselor/commit/b47e8661555bdb751706ae7e772fbda325c05049

File: db/seeds.rb (hunk 2 above)
Comment (travisvalentine) [reply db/seeds.rb:14]: Fixed in https://github.com/tpitale/counselor/commit/b47e8661555bdb751706ae7e772fbda325c05049

**Comment by travisvalentine** [comment]
Closing this since I accidentally pushed this to master :(

**Pull request closed without merging**